octobot_tentacles_manager/exporters/artifact_exporter.py

```diff
@@ -167,3 +169,13 @@ def zip_temporary_folder(self) -> str:
         except Exception as e:
             self.logger.error(f"Error when cleaning up temporary folder: {e}")
         return zipped_file
+
+    def get_exported_file_name(self) -> str:
+        """
+        :return: the export file name depending on compilation
+        """
+        if self.should_cythonize:
+            return f"{util.get_os_str()}_{util.get_arch_str()}"
+        if self.use_package_as_file_name:
```

Comment:
we should add in the cli doc that the cythonized option makes use_package_as_file_name unusable

Reply:
I don't understand why

Reply:
if we ask to cythonize the tentacles, then the package name witll not contain `self.artifact.name` even if `use_package_as_file_name` is true

Reply:
oh yes, my bad. I think `if self.use_package_as_file_name:` should be the first condition.

Reply:
Then we should add a not to the doc saying that the name will not contain the os data if  `use_package_as_file_name` is provided (which is logical but worth adding in my opinion)

Reply:
:+1: